examples/methods/methods.go

```diff
@@ -6,19 +6,28 @@ import "fmt"
 
 type rect struct {
 	width, height int
+	label         string
 }
 
-// This `area` method has a _receiver type_ of `*rect`.
+// This `area` method has a _pointer receiver type_ `*rect`.
 func (r *rect) area() int {
 	return r.width * r.height
 }
 
 // Methods can be defined for either pointer or value
-// receiver types. Here's an example of a value receiver.
+// receiver types. This method `perim` of _value receiver type_
+// will pass the `rect` struct by value instead of reference for
+// each call.
 func (r rect) perim() int {
 	return 2*r.width + 2*r.height
 }
 
+// The method `setLabel` will modify the rect's (pointer receiver)
```

Comment:
Maybe

"`setLabel`  sets the label field to `label`. It has to be defined with a pointer receiver."